README.md

```diff
@@ -935,6 +935,14 @@ your application.
     # more settings
   }
   ```
+  
+* <a name="letter-opener"></a>
```

Comment:
I think @bbatsov meant to write Email rather than Emacs.

@juanpastas: [ActionMailer previews](http://guides.rubyonrails.org/4_1_release_notes.html#action-mailer-previews)